module/Api/src/Domain/Repository/Task.php

```diff
@@ -139,10 +139,10 @@ public function flagUrgentsTasks()
     {
         /** @var \Doctrine\DBAL\Result $stmt */
         $stmt = $this->getDbQueryManager()
-            ->get('Task/FlagUrgentTasks')
+            ->get('Task\FlagUrgentTasks')
             ->execute();
 
-        return $stmt->fetchColumn(0);
+        return (int) $stmt->fetchOne();
```

Comment:
$stmt->rowCount() might be more appropriate here. No need for the cast to Int that way either.

Reply:
the SP does a rowcount and returns that value as a column, so rowCount always equals 1 so we do need to fetchOne()